### ISSUE TITLE
Change /info HTML header

### ISSUE THREAD

**Issue by CasperWA** [Materials-Consortia/providers#1]
If possible, it would be nice to change the `/info` `index.html` to have the HTML header `Content-Type: application/json`.

Since this doesn't change the content, it is not essential.

**Comment by rartino** [comment]
But, do you have any idea how to do that when serving out of github-pages?

Do you remember Materials-Consortia/OPTiMaDe#72? That was exactly because of the difficulty of handling these things in static hosting. We probably should get the fix for that issue merged.

**Comment by CasperWA** [comment]
> But, do you have any idea how to do that when serving out of github-pages?

It seems to work for the symlinked `/links/index.html`.
So an easy fix would be to create the json file in the `info` folder and do the same, i.e., link up `index.html` to the json.
Otherwise we should actually write some HTML in the `index.html` file I guess, adding the header manually.

> Do you remember [Materials-Consortia/OPTiMaDe#72](https://github.com/Materials-Consortia/OPTiMaDe/issues/72)? That was exactly because of the difficulty of handling these things in static hosting. We probably should get the fix for that issue merged.

Touché :)

**Comment by rartino** [comment]
> It seems to work for the symlinked /links/index.html.

How do you get that? I get this:
```
$ curl -s -i https://www.optimade.org/providers/links/
HTTP/2 200 
server: GitHub.com
content-type: text/html; charset=utf-8
last-modified: Thu, 05 Dec 2019 01:39:05 GMT
...
```
And it looks exactly the same for `info/`. 

And, besides, `application/json` isn't even the correct MIME-type. According to JSON API it MUST be `application/vnd.api+json`.

> Otherwise we should actually write some HTML in the index.html file I guess, adding the header manually.

What? How? Are you talking about the html meta http-equiv attribute? If you add html to the file that is served by the `info/` endpoint, you are going to serve OPTiMaDe clients that html. If you think this can work, please demonstrate how.

**Comment by CasperWA** [comment]
Hmm. Actually doing some proper checking on my part, you're completely right. Sorry.

I have found [this stackoverflow page]()

**Comment by CasperWA** [comment]
Conclusion: Let's put in a fix for Materials-Consortia/OPTiMaDe#72
